<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundSettingsForm</class>
 <widget class="QWidget" name="SoundSettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="generalVolumeSlider">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="effectsVolumeSlider">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="finishedButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>500</y>
     <width>224</width>
     <height>59</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border: none; padding: 0; margin: 0; background: transparent;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
